cpp/src/parquet/file_deserialize_test.cc

```diff
@@ -315,6 +325,127 @@ TEST_F(TestPageSerde, LZONotSupported) {
   ASSERT_THROW(InitSerializedPageReader(data_size, Compression::LZO), ParquetException);
 }
 
+TEST_F(TestPageSerde, CrcEnabled) {
```

Comment:
Do you mean `reader_test.cc`? I think they works in different layer. This test just test if our `Page` works, `reader_test` will check if it can read parquet-mr's file. I think we should keep it here

If I missed something, please tell me

Reply:
Yes, I meant that. But ok to keep these tests as well if you feel they're useful.